docs/content/en/docs/intro/_index.md

```diff
@@ -39,10 +37,19 @@ The Custom Keptn metrics feature extends the functionality of
   as well as data that comes directly from your cloud provider
   such as AWS, Google, or Azure.
 
+* Enhances the Kubernetes
+  [Horizontal Pod Autoscaling](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+  facility.
+
 The Keptn metrics server unifies and standardizes
 access to data from various sources,
 simplifying configuration and integration into a single set of metrics.
 
+To learn more:
+
+* [Custom Keptn metrics](usecase_metrics)
```

Comment:
doesn't this need a `#`?
```suggestion
* [Custom Keptn metrics](usecase_metrics)
```

Reply:
No, usecase_metrics is a separate file under what is now the intro section.  It's a very screwy arrangement that needs to be fixed but, for now, it is what it is